src/views/pages/Login.vue

```diff
@@ -85,6 +85,13 @@ export default {
     ValidationObserver
   },
   data() {
+    let redirectUri = `${ window.location.origin } /static/oidc-callback.html`;
```

Comment:
This produces an invalid URL due to the space between `}` and `/static`.

Reply:
fixed it